apps/design/backend/src/ballot_interpretation.test.ts

```diff
@@ -292,3 +297,90 @@ describe('Laid out ballots', () => {
     ).toEqual(sortVotesDict(votes));
   });
 });
+
+describe('Laid out ballots - electionSample ', () => {
+  const election: Election = {
+    ...electionSample,
+    // Fill in missing mark thresholds
+    markThresholds: electionFamousNames2021Fixtures.election.markThresholds,
+  };
+  // Has ballot measures
+  const ballotStyle = assertDefined(
+    getBallotStyle({ election, ballotStyleId: '5' })
+  );
+  const precinct = assertDefined(
+    getPrecinctById({ election, precinctId: ballotStyle.precincts[0] })
+  );
+
+  test('Blank ballot interpretation', async () => {
+    const ballotResult = layOutBallot(election, precinct, ballotStyle);
+    assert(ballotResult.isOk());
+    const { document: ballot, gridLayout } = ballotResult.ok();
+    // We only support single-sheet ballots for now
+    ballot.pages = ballot.pages.slice(0, 2);
+
+    const [frontResult, backResult] = await interpretBallot({
+      election: { ...election, gridLayouts: [gridLayout] },
+      precinct,
+      ballot,
+    });
+
+    assert(frontResult.interpretation.type === 'InterpretedHmpbPage');
+    expect(frontResult.interpretation.votes).toEqual({});
+    assert(backResult.interpretation.type === 'InterpretedHmpbPage');
+    expect(backResult.interpretation.votes).toEqual({});
+  });
+
+  test('Marked ballot interpretation', async () => {
+    // Since we currently only support interpreting single-sheet ballots, we can
+    // only evaluate interpreting contests that we know will fit on two pages.
+    const contestsOnFirstSheet = assertDefined(
+      getContests({ election, ballotStyle })
+    ).filter((contest) =>
+      [
+        'president',
+        'representative-district-6',
+        'lieutenant-governor',
+        'state-senator-district-31',
+        'state-assembly-district-54',
+        'county-registrar-of-wills',
+        'judicial-robert-demergue',
+        'question-a',
+        'question-b',
+      ].includes(contest.id)
+    );
+    const votes: VotesDict = Object.fromEntries(
+      contestsOnFirstSheet.map((contest, i) => {
+        if (contest.type === 'candidate') {
+          const candidates = range(0, contest.seats).map(
+            (j) => contest.candidates[(i + j) % contest.candidates.length]
+          );
+          return [contest.id, candidates];
+        }
+        return [contest.id, i % 2 === 0 ? ['yes'] : ['no']];
+      })
+    );
+
+    const ballotResult = layOutBallot(election, precinct, ballotStyle);
+    assert(ballotResult.isOk());
+    const { document: ballot, gridLayout } = ballotResult.ok();
+    // We only support single-sheet ballots for now
```

Comment:
The interpreter can't scan multiple sheets because we don't have a way to encode the sheet number in the ballot metadata (timing marks). We can and do layout ballots with multiple sheets, they just aren't interpretable.

Reply:
This will be fixed when we move to a QR code for metadata